src/main/fc/runtime_config.c

```diff
@@ -23,19 +23,19 @@
 #include "fc/runtime_config.h"
 #include "io/beeper.h"
 
-uint8_t armingFlags = 0;
-uint8_t stateFlags = 0;
-uint16_t flightModeFlags = 0;
+armingFlag_e armingFlags = 0;
+stateFlags_e stateFlags = 0;
+flightModeFlags_e flightModeFlags = 0;
```

Comment:
I think it is fine to use enums rather than defined constants for the actual values. It's the use of the enum type rather than uintN_t for storing the bits that I think is wrong.